src/hooks/useTimestamp.ts

```diff
@@ -0,0 +1,17 @@
+import { useState, useEffect } from 'react'
+
+export function useTimestamp(interval = 1_000): number {
```

Comment:
I would call this hook `useClock` or `useIntervalTimestamp`. `useTimestamp` to me reads as a one-time timestamp.

Reply:
I've renamed it to `useClock` in 6896ea6497e6318c8ffb226c27737f612c12f456.